core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -417,7 +432,7 @@ public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
     }
 
     @Override
-    public void start() {
+    public void invoke() {
```

Comment:
that's a good change, I was confused by `getDockerCompose(...).start(...)` constructions :)
P.S. do we use anything else besides `invoke()` on `getDockerCompose`'s return value? Is not, maybe we can simplify it to something like `runWithCompose("down -v")` ?